lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -31,7 +31,7 @@
 
 DEFAULT_BASE_PATH_DIRECTORY = "_conda"
 DEFAULT_CONDARC_OVERRIDE = "_condarc"
-DEFAULT_ENSURE_CHANNELS = "conda-forge,r,bioconda,iuc"
+DEFAULT_ENSURE_CHANNELS = "iuc,bioconda,r,defaults,conda-forge"
```

Comment:
Sorry for being late to the game, but is it possible that this list should be reversed?
It appears that currently conda-forge takes the lowest priority.
Admittedly this is a bit confusing, but it appears that https://github.com/galaxyproject/tools-devteam/pull/375 fails because bioconda takes the highest priority, while conda-forge should be highest. This also explains passing tests in bioconda but failing when we try to use them in galaxy.